permissions/plugin-configuration-as-code-secret-ssm.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "configuration-as-code-secret-ssm"
+github: "jenkinsci/configuration-as-code-secret-ssm-plugin"
+paths:
+- "com/bambora/jenkins/plugin/casc/secrets/ssm/configuration-as-code-secret-ssm"
```

Comment:
I like how the groupId duplicates basically the entire artifactId, but I'm weird 😆